src/mlpack/tests/main_tests/logistic_regression_test.cpp

```diff
@@ -103,7 +103,7 @@ BOOST_AUTO_TEST_CASE(LRPridictionSizeCheck)
   mlpackMain();
 
   // Get the output predictions of the test data.
-  const arma::Row<size_t> &testY = CLI::GetParam<arma::Row<size_t>>("output");
+  const arma::Row<size_t> &testY = CLI::GetParam<arma::Row<size_t>>("predictions");
```

Comment:
Could you elaborate on why the modification of the call to ```CLI::GetParam<arma::mat>``` ?

Reply:
I'd suggest trying to run the `mlpack_logistic_regression` binding before and after your changes, to see how the `--output_file` and `--predictions_file` options behave.  This will make it clear why the changes are necessary (i.e., the existing code will not properly set the `predictions` output).

Reply:
Okay sure

Reply:
In your code the `predictions` output parameter is not being set with `CLI::GetParam`.  Take a look at the existing code, other command line programs, and the documentation for the CLI system and hopefully this should clarify the changes that need to be made. :+1: